src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -25,4 +43,35 @@ public SearchIndexModel indexSnapshot(Snapshot snapshot, SearchIndexRequest sear
         List<Map<String, Object>> values = bigQueryPdao.getSnapshotTableData(snapshot, searchIndexRequest.getSql());
         return new SearchIndexModel();
     }
+    public SearchQueryResultModel querySnapshot(
+            SearchQueryRequest searchQueryRequest, List<String> indicesToQuery,
+            @Valid Integer offset, @Valid Integer limit
+    ) {
+        //todo: move to bean for configuration, make injectable
+        final RestClientBuilder builder = RestClient.builder( new HttpHost("localhost", 9200));
+        try (RestHighLevelClient client = new RestHighLevelClient(builder)) {
+            SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
```

Comment:
This is interesting, I noticed that the bulk api Florian is using has both a builder and a non-builder version. Did you happen to notice two styles of apis for this as well?